firefox-ios/Client/Utils/OhttpGleanUploader.swift

```diff
@@ -0,0 +1,92 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import Glean
+
+/// An enumeration representing different environments for the OHTTP client.
+public enum OhttpEnvironment {
+    case staging
+    case prod
+
+    // TODO: Laurie - config URL to pass into the OHTTPManager?
+    /// Returns the configuration URL based on the selected environment.
+    var config: URL? {
+        switch self {
+        case .staging:
+            return URL(string: "https://stage.ohttp-gateway.nonprod.webservices.mozgcp.net/ohttp-configs")
+        case .prod:
+            return URL(string: "https://prod.ohttp-gateway.prod.webservices.mozgcp.net/ohttp-configs")
+        }
+    }
+
+    // TODO: Laurie - relay URL to pass into the OHTTPManager?
+    /// Returns the relay URL based on the selected environment.
+    var relay: URL? {
+        switch self {
+        case .staging:
+            return URL(string: "https://mozilla-ohttp-dev.fastly-edge.com/")
+        case .prod:
+            return URL(string: "https://mozilla-ohttp-dev.fastly-edge.com/")
+        }
+    }
+
+    static func getEnvironment() -> OhttpEnvironment {
+        // TODO: Laurie - Double-check this is accurate
+        let shouldUseRelease = AppConstants.buildChannel == .release || AppConstants.buildChannel == .beta
+        return shouldUseRelease ? OhttpEnvironment.prod: OhttpEnvironment.staging
+    }
+}
+
+public struct OhttpGleanUploader: PingUploader {
+    private let environment: OhttpEnvironment
+    private let capabilities = ["ohttp"]
+    private let connectionTimeout = TimeInterval(10)
```

Comment:
I just realized Glean as a [connectionTimeout of 10 000 seconds](https://github.com/mozilla/glean/blob/28f0ab611cbbc1160eeb520c1eb45d9eae44340e/glean-core/ios/Glean/Net/HttpPingUploader.swift#L18), was that the intent? It's used as `TimeInterval(Constants.connectionTimeout) and `TimeInterval` is in seconds.

I set 10 seconds here for the `connectionTimeout`.